numpyro/infer/autoguide.py

```diff
@@ -1134,6 +1138,7 @@ def local_model(theta):
     :param callable global_guide: A guide for the global latent variables, e.g. an autoguide.
         The return type should be a dictionary of latent sample sites names and corresponding samples.
         If there is no global variable in the model, we can set this to None.
+    :param callable local_guide: An optional guide for the local latent variables.
```

Comment:
`An optional guide for specifying the DAIS base distribution for local latent variables` ?

Reply:
done